Code of conduct updates, clarifications, and revisions (#13)
beyond-all-reason/code-of-conduct: CodeOfConduct.md

g# Overview

> We like to keep the BAR ecosystem as fun and welcoming as possible. As such, we prepared a simple set of rules to regulate our online community. These rules apply to all games and chat hosted on official BAR servers.

Reviewer: The original sounds better here imo, and might be more technically correct.

> All of us are responsible for creating and contributing to a positive community and all of us can be held accountable so please take the time to read this. You will be held accountable for your actions regardless of whether have read this. 

Reviewer: This seems grammatically incorrect?

Reviewer 2: > whether have read this.

Correction: whether you have read this or not.

> At the root of our expectation is for players to be civil and nice to each other; of course there is room for competitiveness and we don't mind friendly banter but in short, be nice to each other. People disturbing or not respecting this will be reminded where needed. We reserve the right to suspend your account for any committed infraction detailed below. 

Reviewer: > At the root of our expectation

What expectation? I don't like this whole paragraph tbh.

...oops, it was there originally. Maybe simplify it to "We expect players to be civil and nice to each other"?

## **What to do** when you see this code of conduct not being followed
We have only so many moderators and despite the large amounts of ☕ they consume they cannot be everywhere at once. There are three steps you can take when you see something inappropriate, depending on how severe the issue is depends on which step you can take:

> + **Report it:** If something is bad enough that it needs to be reported, you can report ingame by right-clicking on players name, and choosing "report" from the dropdown. Alternatatively, you can submit the report using the following link: https://server4.beyondallreason.info/battle/ and finding the relevant match, then clicking the "players" tab and reporting the user from there.

Reviewer: ```suggestion
+ **Report it:** If something is bad enough that it needs to be reported, you can report ingame by right-clicking on player's name, and choosing "report" from the dropdown. Alternatively, you can submit the report using the following link: https://server4.beyondallreason.info/battle/ and finding the relevant match, then clicking the "players" tab and reporting the user from there.
```

+ In case of having an active griefer in a game, **don't retaliate with force**. Instead, use some of available the self-moderation tools. Note that retaliating against griefing may result in moderation actions being taken against you. If you believe it is necessary to prevent someone from persisting in their abusive behavior, try:
    + _pausing the game (Pause|Break button or /pause command)_
    + `!kickban <playername>` _- calls a vote to kick and temporarily ban the player from the lobby _

>     + _make sure to explain it in chat so that others know why they should vote him out_

Reviewer: ```suggestion
    + _make sure to explain it in chat so that others know why they should vote them out_
```

    + _in case of verbal harassment or spamming you can also call a_ `!mute` _vote_

> + **Offer support:** Stick up for people being bullied. Publicly calling out bad behavior and privately messaging the person in question and giving them support are examples of ways you can stick up for people being bullied.

Reviewer: That sounds like an invitation to a chat war on someone's behalf, should we keep this?

+ **Mute individually** - you can mute individual players by right-clicking on their name in lobby and choosing "ignore" from the dropdown menu. To mute ingame, ctrl+left click on their name in the player list. Note that ingame and lobby mutes are different and separate.

## **Section A** - Be nice to each other
This section contains things which are less specific and often less problematic in small doses; they're also things people may do by mistake. While they are reportable in many cases, communication and education will bring about a better result.

### 1. We are a community
1. Respect for all other members is mandatory.
2. Harassment and baiting of any community member, including developers and contributors, is not acceptable.

> 3. Intentionally making false accusations, for example with the intent to have a player removed from the game/lobby, is abusive. We understand that mistakes happen, and no one knows everything that goes on in each match they play.

Reviewer: That sounds like an actual rule change, are we ok with this?

### 2. Harassment or bullying
1. Saying "it was just a joke" does not excuse this either; it's only a joke if all parties find it funny (including 3rd parties if it’s in a public situation).
2. Harassment includes flaming players for using strategies which may seem subjectively suboptimal (high-risk tactics like commander-drops and such) or just low skill plays.

### 3. Team games
1. Disregarding the teamwork part of team games when done notoriously can result in being ostracized from team games.
2. You don't have to do everything an ally says, but at the same time you are part of the team; being a good team player includes giving teammates an appropriate amount of space to build and respecting their claim to the available resources on the map.

### 4. Spoiling games
1. Dragging out clearly won games is poor form and undesirable.
2. Players may leave games for a variety of valid reasons. Players who make a habit of leaving ongoing matches long before they end risk being met with moderation action. Matches thought to suffer from external issues suchs as bugs require a restart and can be stopped by calling a stop vote with `!cv stop` in chat.

> 3. Pausing with the intent to disrupt the game is not acceptable.

Reviewer: original phrasing is more clear imo

> 4. Purposefully unpausing to disrupt a player reconnecting or to disrupt a discussion/vote to kickban a misbehaving player, unless a reasonable amount of time has passed, is unsportsmanlike and to be avoided.

Reviewer: ```suggestion
4. Purposefully unpausing to disrupt a pause issued for a legitimate reason, or without taking steps to find out the reason, unless a reasonable amount of time has passed, is unsportsmanlike and to be avoided. Legitimate reasons include, but aren't limited to: discussion/vote to kickban a misbehaving player, players reconnecting, lag or in-game bugs.
```

These are much clearer cut rules and reporting is a completely acceptable first response. If a moderator sees one of these take place they will likely use a formal warning as a starting point.

### 1. Inappropriate topics
1. It is not okay to promote illegal or damaging topics including but not limited to gambling, drug use, self-harm, crypto scams, and the like.

> 2. Politics, sexually explicit chat, and advertisements are examples of other inappropriate topics - please keep BAR chat related to BAR!

Reviewer: ```suggestion
2. Politics, sexually explicit chat, and advertisements are examples of other inappropriate topics.
```

Players are free to discuss cookie recipes no matter how unrelated to BAR

3. Some of these can be discussed in the appropriate locations but this is a game and generally not a place for sharing these topics.

### 2. Discrimination
1. Discriminating against someone for who/what they are is not acceptable. Discrimination can take the form of speech but also actions (e.g. kicking them from games).

> 2. It is completely acceptable to remove players from game as a response to problematic behavior but be careful not to cross the line to vigilantism.

Reviewer: ```suggestion
2. It is completely acceptable to remove players from a game as a response to problematic behavior but be careful not to cross the line to vigilantism.
```

Reviewer: The vigilantism comment doesn't make sense if you limit it to kickbanning, maybe this whole line needs a rewrite

> 3. Abuse especially on the grounds or context of gender, race, orientation, disability, religion, ancestry, nationality, and similar is never acceptable.

Reviewer: ```suggestion
3. Abuse especially on the grounds or context of gender, race, sexual orientation, disability, religion, ancestry, nationality, and similar is never acceptable.
```

### 3. Offensive names
1. Nicknames and clan tags must not be offensive, inappropriate or contain politically charged phrases.
2. Impersonation of other players and real-life figures is also forbidden.
3. Player usernames must not contain references to modern political figures, contemporary movements, or historically controversial figures.

> 4. An account which doesn't abide by this rule will be suspended until it is renamed.

Reviewer: Original is better imo

### 4. Griefing

> 1. Griefing, trolling, or actively working against your team is against the rules.

Reviewer: Absolutely not, "trolling" is a nebulous term that we can neither define nor action

2. Examples of griefing include teamkilling, purposefully hurting/imprisoning, capturing or reclaiming allied units without consent or consistently throwing games with the intent to deny other team members a potential win.
3. Griefing does not include reclaiming or reviving wreckage/debris on the map, regardless of where it is located or which player owned the original unit/structure.
   - You should, however, avoid reclaiming wrecks of allied commanders self-destructed to boost a tech rush. *If on a map with plentiful energy reclaim trees, start your reclaim circle on a tree to only reclaim energy and avoid unfortunate accidents.*

> 4. If you are done with a game then resign; self-destructing units to deny them from your team is a form of griefing.

Reviewer: ```suggestion
4. If you are done with a game then resign; self-destructing units to deny them to your own team is a form of griefing.
```

Reviewer 2: If you are finished with a game, then resign; self-destructing units to prevent your own team from using them is a type of griefing.

### 5. Unfair Advantages
1. Custom widgets used on public servers must be made publicly available on the offical BAR discord channel [#widgets](https://discord.com/channels/549281623154229250/1113845509891829810).
2. Purposefully passing on consequential information to a game in progress via things like spectator chat (we understand mistakes happen occasionally) is obviously a form of cheating.

> 3. Pausing the game for tactical purposes is a form of cheating.

Reviewer: Unnecessary, one pause rule is enough
